schemas/Clinical/index.js

```diff
@@ -44,6 +45,13 @@ const typeDefs = gql`
     alreadyRegistered: ClinicalRegistrationStats!
   }
 
+  type ClinicalRegistrationInvalid {
+    error: String
+    code: String
+  }
```

Comment:
to make UI life easier, probably a `programShortName: String` can help :) same with `ClinicalRegistrationData`

Reply:
added `shortName` to both clinical responses

Reply:
I feel `shortName` is a little vague, what do you think of `programShortName`? :)

Reply:
@hlminh2000 that works. to be clear, when do we use `shortName` and when do we use `programShortName`.

ending up pulling shortname from args
``` javascript
const {shortname} = args;
```
then renaming

Reply:
just cause the object type is `ClinicalRegistrationInvalid`, `shortName` sounds like `clinicalRegistrationInvalidShortName`, which isn't a thing :P just my feeling honestly

Reply:
I see your thinking